web/src/LogPane.tsx

```diff
@@ -121,9 +127,13 @@ class LogPane extends Component<LogPaneProps, LogPaneState> {
     }
   }
 
+  isElement(n: any): n is Element {
+    return (n as Element).getAttribute !== undefined
```

Comment:
Is there a better way to do this?

Reply:
this seems wrong. what are you trying to accomplish?